checkbox-ng/checkbox_ng/launcher/master.py

```diff
@@ -434,6 +436,8 @@ def register_arguments(self, parser):
         parser.add_argument(
             "-u", "--user", help=_("normal user to run non-root jobs")
         )
+        parser.add_argument("-n", "--dry-run", default=False, action="store_true", help=_(
+            "don't actually do anything"))
```

Comment:
This description doesn't convey what happens. The resource jobs still run, the relationship between jobs are still computed, and interactive jobs run normally.

More adequate description I guess would be "don't run commands associated with normal jobs".